.github/workflows/ci_infrastructure.yml

```diff
@@ -35,11 +35,15 @@ jobs:
       licenceplate: ${{ secrets.LICENCEPLATE}}
       tfc_team_token: ${{ secrets.TFC_TEAM_TOKEN }}
       dev_oidc_idir_idp_client_secret: "${{ secrets.DEV_OIDC_IDIR_IDP_CLIENT_SECRET }}"
-      test_oidc_bceid_business_idp_client_secret: "${{ secrets.TEST_OIDC_IDIR_IDP_CLIENT_SECRET }}"
```

Comment:
The values here don't actually get used because there is no deployment going on. I fixed them because I figured someone would see that they were different from the other ones and get confused.

Reply:
Just wondering I still saw this variable in the "infrastructure/server/oidc_idp_bceid_business.tf" file line 36, could we confirm that it's not in use? thanks
https://github.com/bcgov/nr-forests-access-management/blob/b0db767334e260124cf3983e6e803ce312f7561e/infrastructure/server/oidc_idp_bceid_business.tf#L36

Reply:
oh I saw your reorder the variables, and added test_oidc_bceid_business_idp_client_secret back in line 41, please ignore my previous comment